### ISSUE TITLE
WebAuthn flow throws error when attempting to register Yubikey

### ISSUE THREAD

**Issue by swishkin** [goauthentik/authentik#5599]
**Describe the bug**
WebAuthn registration fails for Yubikey on Windows using Brave.

> Error: Error creating credential: NotAllowedError: The operation either timed out or was not allowed. See: [https://www.w3.org/TR/webauthn-2/#sctn-privacy-considerations-client](https://www.w3.org/TR/webauthn-2/#sctn-privacy-considerations-client).

**To Reproduce**

1. Go to the admin interface
2. Click on Flows & Stages
3. Click on Stages
4. Locate and click the WebAuthn stage
5. Click "Normal"
6. Select "Windows Hello or external security key" in resulting popup
7. Click Ok on Windows Security Key setup prompt
8. Insert Yubikey in to USB port, enter pin if any is configured/requested
9. Touch security key

Result:

![image](https://github.com/goauthentik/authentik/assets/19404743/38cb4f4b-e4b6-4d32-9dfa-7cc1d0a74af5)

**Expected behavior**
When following the steps above, a confirmation should occur indicating that the WebAuthn method has been registered and can now be used.

**Screenshots**

![image](https://github.com/goauthentik/authentik/assets/19404743/222b2ca7-bbf4-4fd9-bed3-43818974b933)
![image](https://github.com/goauthentik/authentik/assets/19404743/4e62514b-d3cf-4eb3-aa98-8ca915fc4baf)
![image](https://github.com/goauthentik/authentik/assets/19404743/b389ce6d-7463-486e-8381-ffcefc6802cf)
![image](https://github.com/goauthentik/authentik/assets/19404743/67ec0254-01a9-42e3-913f-6d4d77313b7b)
![image](https://github.com/goauthentik/authentik/assets/19404743/3e075bab-5fbd-4839-a12e-ebc3cbff479f)

**Logs**

(domain, user, and IP have been changed for privacy)

> INF undefined | auth_via=unauthenticated event=/ host=authentik.example.com logger=authentik.asgi method=GET pid=30 remote=2001:db8::8a2e:370:7334request_id=34b6d4e11df04160bcf042a0c965b34c runtime=9 scheme=https status=302 timestamp=2023-05-13T20:16:29.824884 user= user_agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.42 
INF undefined | auth_via=unauthenticated event=/flows/-/default/authentication/?next=/ host=authentik.example.com logger=authentik.asgi method=GET pid=30 remote=2001:db8::8a2e:370:7334request_id=3368b307ffd34def8573e14c14147210 runtime=11 scheme=https status=302 timestamp=2023-05-13T20:16:29.958936 user= user_agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.42 
INF undefined | auth_via=unauthenticated event=/if/flow/default-authentication-flow/?next=%2F host=authentik.example.com logger=authentik.asgi method=GET pid=30 remote=2001:db8::8a2e:370:7334request_id=bc02c0eb28834b03a7e951160cce455a runtime=17 scheme=https status=200 timestamp=2023-05-13T20:16:30.108075 user= user_agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.42 
INF undefined | auth_via=unauthenticated event=/api/v3/flows/executor/default-authentication-flow/?query=next%3D%252F host=authentik.example.com logger=authentik.asgi method=GET pid=30 remote=2001:db8::8a2e:370:7334request_id=259a8d60294b4c7c82cb452cd1d41156 runtime=47 scheme=https status=200 timestamp=2023-05-13T20:16:30.657823 user= user_agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.42 
INF undefined | auth_via=unauthenticated event=/api/v3/flows/executor/default-authentication-flow/?query=next%3D%252F host=authentik.example.com logger=authentik.asgi method=POST pid=30 remote=2001:db8::8a2e:370:7334request_id=1e1feb2bfdfe44ab97e33897142da897 runtime=21 scheme=https status=302 timestamp=2023-05-13T20:16:32.174961 user= user_agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.42 
INF undefined | auth_via=unauthenticated event=/api/v3/flows/executor/default-authentication-flow/?query=next%3D%252F host=authentik.example.com logger=authentik.asgi method=GET pid=30 remote=2001:db8::8a2e:370:7334request_id=abcf5fc882074e5fb8bd53c638af265e runtime=35 scheme=https status=200 timestamp=2023-05-13T20:16:32.262168 user= user_agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.42 
INF undefined | auth_via=unauthenticated backend=authentik.core.auth.InbuiltBackend event=Successful authentication host=authentik.example.com logger=authentik.stages.password.stage pid=29 request_id=c046b03860e44fd482539bcb1eb22e00 timestamp=2023-05-13T20:16:38.284000 user=user 
INF undefined | auth_via=unauthenticated event=/api/v3/flows/executor/default-authentication-flow/?query=next%3D%252F host=authentik.example.com logger=authentik.asgi method=POST pid=29 remote=2001:db8::8a2e:370:7334request_id=c046b03860e44fd482539bcb1eb22e00 runtime=314 scheme=https status=302 timestamp=2023-05-13T20:16:38.288389 user= user_agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.42 
INF undefined | auth_via=unauthenticated event=/api/v3/flows/executor/default-authentication-flow/?query=next%3D%252F host=authentik.example.com logger=authentik.asgi method=GET pid=29 remote=2001:db8::8a2e:370:7334request_id=ef1c67699ab3463cacd5ae923ec878da runtime=2060 scheme=https status=200 timestamp=2023-05-13T20:16:40.401952 user= user_agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.42 
INF undefined | auth_via=unauthenticated event=/-/health/live/ host=localhost:8000 logger=authentik.asgi method=GET pid=28 remote=127.0.0.1 request_id=f4961ed09c444941a36de21f19fff0bf runtime=9 scheme=http status=204 timestamp=2023-05-13T20:16:46.865266 user= user_agent=goauthentik.io/proxy/healthcheck 
INF undefined | auth_via=unauthenticated event=/-/health/ready/ host=localhost:9000 logger=authentik.asgi method=HEAD pid=28 remote=127.0.0.1 request_id=528be60da9ff4e47925909758f2c8902 runtime=9 scheme=http status=204 timestamp=2023-05-13T20:16:46.978319 user= user_agent=goauthentik.io lifecycle Healthcheck 
INF undefined | auth_via=unauthenticated event=/api/v3/flows/executor/default-authentication-flow/?query=next%3D%252F host=authentik.example.com logger=authentik.asgi method=POST pid=28 remote=2001:db8::8a2e:370:7334request_id=ebe229df41234c96b58c225d42123ad1 runtime=41 scheme=https status=302 timestamp=2023-05-13T20:17:01.293136 user= user_agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.42 
INF undefined | auth_via=unauthenticated event=/api/v3/flows/executor/default-authentication-flow/?query=next%3D%252F host=authentik.example.com logger=authentik.asgi method=GET pid=28 remote=2001:db8::8a2e:370:7334request_id=8d882629cc594e768036390ee7464edf runtime=21 scheme=https status=200 timestamp=2023-05-13T20:17:01.364566 user= user_agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.42 
INF undefined | auth_via=unauthenticated event=/-/health/live/ host=localhost:8000 logger=authentik.asgi method=GET pid=29 remote=127.0.0.1 request_id=466e8925cb1d4c0f9bbb4590e2541374 runtime=9 scheme=http status=204 timestamp=2023-05-13T20:17:16.865128 user= user_agent=goauthentik.io/proxy/healthcheck 
INF undefined | auth_via=unauthenticated event=/-/health/ready/ host=localhost:9000 logger=authentik.asgi method=HEAD pid=29 remote=127.0.0.1 request_id=90c6bfa769e7495791fa03054a7ed87a runtime=9 scheme=http status=204 timestamp=2023-05-13T20:17:17.098289 user= user_agent=goauthentik.io lifecycle Healthcheck 
INF undefined | auth_via=unauthenticated event=/-/health/live/ host=localhost:8000 logger=authentik.asgi method=GET pid=29 remote=127.0.0.1 request_id=9132e2d67faf4da0bcf8ede813032008 runtime=11 scheme=http status=204 timestamp=2023-05-13T20:17:46.902207 user= user_agent=goauthentik.io/proxy/healthcheck 
INF undefined | auth_via=unauthenticated event=/-/health/ready/ host=localhost:9000 logger=authentik.asgi method=HEAD pid=29 remote=127.0.0.1 request_id=5f2745e2a22145b48883f35bef961131 runtime=10 scheme=http status=204 timestamp=2023-05-13T20:17:47.202697 user= user_agent=goauthentik.io lifecycle Healthcheck 
INF undefined | event=updating tenant certificates logger=authentik.router.tenant_tls timestamp=2023-05-13T20:17:47Z 
INF undefined | auth_via=unauthenticated event=Task published host=localhost:8000 logger=authentik.root.celery pid=30 request_id=7c526726621e426c85281e690ef8eab1 task_id=6eef14e0-8650-4284-8a51-f61977befbbd task_name=authentik.providers.scim.tasks.scim_signal_direct timestamp=2023-05-13T20:17:47.872841 
INF undefined | auth_via=secret_key event=/api/v3/core/tenants/ host=localhost:8000 logger=authentik.asgi method=GET pid=30 remote=127.0.0.1 request_id=7c526726621e426c85281e690ef8eab1 runtime=37 scheme=http status=200 timestamp=2023-05-13T20:17:47.877947 user=ak-outpost-b2ee18e3e7c54c909e5ea2af9ee066e0 user_agent=goauthentik.io/outpost/2023.4.1 
INF undefined | auth_via=unauthenticated event=/-/health/live/ host=localhost:8000 logger=authentik.asgi method=GET pid=29 remote=127.0.0.1 request_id=cb6a234014bd4a5ba7ee2fdd79edc667 runtime=8 scheme=http status=204 timestamp=2023-05-13T20:18:16.865606 user= user_agent=goauthentik.io/proxy/healthcheck 
INF undefined | auth_via=unauthenticated event=/-/health/ready/ host=localhost:9000 logger=authentik.asgi method=HEAD pid=29 remote=127.0.0.1 request_id=13442d4456f0492a9d512c11a69303ef runtime=9 scheme=http status=204 timestamp=2023-05-13T20:18:17.301928 user= user_agent=goauthentik.io lifecycle Healthcheck 
INF undefined | auth_via=unauthenticated event=/-/health/live/ host=localhost:8000 logger=authentik.asgi method=GET pid=30 remote=127.0.0.1 request_id=3e03b9e847a24e96994b9ed1e8fc6db4 runtime=9 scheme=http status=204 timestamp=2023-05-13T20:18:46.865757 user= user_agent=goauthentik.io/proxy/healthcheck 
INF undefined | auth_via=unauthenticated event=/-/health/ready/ host=localhost:9000 logger=authentik.asgi method=HEAD pid=30 remote=127.0.0.1 request_id=166c86fa83444ba2b458203637146ff6 runtime=9 scheme=http status=204 timestamp=2023-05-13T20:18:47.394351 user= user_agent=goauthentik.io lifecycle Healthcheck 
INF undefined | auth_via=unauthenticated event=/-/health/live/ host=localhost:8000 logger=authentik.asgi method=GET pid=30 remote=127.0.0.1 request_id=be98177c348d4d488651b9d15e5570ea runtime=8 scheme=http status=204 timestamp=2023-05-13T20:19:16.865031 user= user_agent=goauthentik.io/proxy/healthcheck 
INF undefined | auth_via=unauthenticated event=/-/health/ready/ host=localhost:9000 logger=authentik.asgi method=HEAD pid=30 remote=127.0.0.1 request_id=a667b7deef4244fcadc2495177cf7842 runtime=9 scheme=http status=204 timestamp=2023-05-13T20:19:17.498411 user= user_agent=goauthentik.io lifecycle Healthcheck 

**Version and Deployment (please complete the following information):**

-   authentik version: 2023.4.1
-   Deployment: Portainer Stack

**Additional context**

I'm using Brave v1.51.114, but tested with Edge 113.0.1774.42 and got the same result.

I searched around for guides to setting up Yubikey WebAuthn in Authentik, but haven't found other issues or guides.

**Comment by jonade** [comment]
Interesting. I've got the same issue, and with the same browsers (Brave + Edge). Issue remains when testing with both a Yubikey5 and a Nano 5C

**Comment by tograss** [comment]
For me it works. I tested chrome and Firefox. I also used the YubiKey security key, which might be the discriminating factor.

**Comment by swishkin** [comment]
Yeah, I forgot to mention I'm using a Yubikey 5 NFC (USB A); if others don't have the issue with the same model, that would be curious.

**Comment by swishkin** [comment]
I tried the same process described in my initial post, except with Brave v1.52.122 and Authentik 2023.5.3; I received the same result.

**Comment by swishkin** [comment]
So, I tested a few things.

Enrolling my primary Yubikey on my preferred browser (Brave) on either of my computers (Windows 10 desktop, Windows 10 laptop) fails as described.

Enrolling a different Yubikey works in all Windows settings I tested.

Enrolling my primary Yubikey on another Windows PC (regardless of browser) works as expected.

Initially, I thought it might be the limited quantity of Webauthn keys Yubikeys can store:

> FIDO2 - the YubiKey 5 can hold up to 25 discoverable credentials in its FIDO2 application.
[from here](https://support.yubico.com/hc/en-us/articles/360013790319-How-many-accounts-can-I-register-my-YubiKey-with-)

However, when I run ykman fido credential list, it only shows two credentials, suggesting I should have 23 remaining.

Further, even if the issue _were_ no more WebAuthn/FIDO slots available, it wouldn't explain why my key worked perfectly to enroll in WebAuthn on another Windows PC.
